AppCenter/AppCenterTests/MSAppCenterTests.m

```diff
@@ -465,8 +465,8 @@ - (void)testDisableServicesWithEnvironmentVariable {
   [MSAppCenter start:@"AppSecret" withServices:@[ MSMockService.class, MSMockSecondService.class ]];
 
   // Then
-  XCTAssertFalse([[MSMockService sharedInstance] started]);
-  XCTAssertFalse([[MSMockSecondService sharedInstance] started]);
+  XCTAssertFalse([MSMockService sharedInstance].started);
```

Comment:
Nit, since those lines are marked as changed (probs not for this PR, but still) - we now would have designated getters, we should probably use them instead direct access.

Reply:
So we need to use the getter instead of direct access, right? @clpolet , can you help to confirm it?